lib/widget/earthquake_report_list_tile.dart

```diff
@@ -35,23 +35,46 @@ class EarthquakeReportListTile extends StatelessWidget {
             ),
           ),
         ),
-        additionalInfo: Container(
-          width: 48,
-          height: 48,
-          decoration: BoxDecoration(
-            borderRadius: BorderRadius.circular(12.0),
-            color: context.colors.intensity(report.intensity),
-          ),
-          child: Center(
-            child: Text(
-              intensityToNumberString(report.intensity),
-              style: TextStyle(
-                fontSize: 28,
-                fontWeight: FontWeight.bold,
-                color: context.colors.onIntensity(report.intensity),
+        additionalInfo: Row(
+          children: [
+            Container(
+              width: 48,
+              height: 48,
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(12.0),
+                color: getMagnitudeColor(report.mag),
+              ),
+              child: Center(
+                child: Text(
+                  "${report.mag}", // 顯示地震規模
+                  style: TextStyle(
+                    fontSize: 28,
```

Comment:
```suggestion
                    fontSize: 24,
```
會溢出的話就把字體調小就好，不要把容器調大，造成不一致性